src/components/Form.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const form = () => (
+  <div>
+    Hi
+  </div>
+);
+
+export default form;
```

Comment:
- [ ] I think that you're not using this component anywhere, so please consider removing it as not used code can pollute your code base and make it harder to read! 🤓 🔧